README.md

```diff
@@ -7,7 +7,7 @@
 To install dependencies, run:
 
 ```
-pip install -r requirements.txt
+pip install -r requirements.txt --use-deprecated=legacy-resolver
```

Comment:
Why do we change instructions also for local installs if the issue appears only in CI or Netlify? 🤔

Reply:
Removed.